### ISSUE TITLE
Check the plugin size

### ISSUE THREAD

**Issue by fredck** [cksource/ckeditor-plugin-a11ychecker#202]
I've just made two builds in the online builder, one with A11y Checker and another without it. The following is the size of `ckeditor.js` in both cases:

 * Without A11y Checker: **518 KB**
 * With A11y Checker: **701 KB** (+193 KB)

In theory, QUAIL and tests should be loaded on demand when clicking on the A11y Checker button. Therefore, almost 200 KB on the plugin size seems to be a bit to excessive. It should be checked what is ending up in the plugin file and if the loading on demand mechanism is working properly.

**Comment by mlewand** [comment]
Yea, I can see Quail is inlined. Without Quail built plugin.js is 43KB compared to 175 KB when Quail is added.

**Comment by wimleers** [comment]
As https://www.drupal.org/node/2731373#comment-11243745 explains, we'd love to include the `a11ychecker` plugin, but we'd need `quail` to be loaded separately.

So, when you implement this on-demand loading, can you please also make it possible to let the site specify the _location_ of the `quail` library? Otherwise sites that have other uses for `quail` will need to ship it twice: once as part of `a11ychecker`, once independently.

**Comment by mlewand** [comment]
> So, when you implement this on-demand loading, can you please also make it possible to let the site specify the location of the quail library?

Yes, it makes a perfect sense. We were thinking about this at the beginning, but for some reason this has slipped out. I've extracted this request to #204.

**Comment by mgifford** [comment]
Looking forward to this! So good to be able to bring accessibility right to the content editors. Big step towards ATAG 2.0 compliance too.
